mypy/test/testcheck.py

```diff
@@ -73,6 +70,9 @@
     'check-class-namedtuple.test',
     'check-selftype.test',
     'check-python2.test',
+    'check-columns.test',
+    'check-functions.test',
+    'check-tuples.test',
```

Comment:
I cannot comment on lines 81 ('check-newsyntax.test') and 84 ('check-underscores.test'), but I think those two could be changed from ``files.append(...)`` to ``fast_parser_files.append(...)``